mem/mem.go

```diff
@@ -72,6 +79,9 @@ func Meta() *plugin.PluginMeta {
 // GetMetricTypes returns list of available metric types
 // It returns error in case retrieval was not successful
 func (mp *memPlugin) GetMetricTypes(cfg plugin.ConfigType) ([]plugin.MetricType, error) {
+	if procPath, ok := cfg.Table()["proc_path"]; ok {
```

Comment:
For the sake of consistency consider using `config.GetConfigItem(cfg, "proc_path")`